x-pack/plugins/security_solution/cypress/tasks/rule_filters.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const expectRulesWithExecutionStatus = (status: string, expectedCount: number) => {
+  cy.get(`[data-test-subj="ruleExecutionStatus"]:contains("${status}")`).should(
```

Comment:
It's a good pattern to have selectors in screens folder, for example this one can go to `x-pack/plugins/security_solution/cypress/screens/alerts_detection_rules.ts`.